visual_behavior/validation/sdk.py

```diff
@@ -0,0 +1,293 @@
+from allensdk.brain_observatory.behavior.behavior_project_cache import BehaviorProjectCache as bpc
+from allensdk.brain_observatory.behavior.behavior_data_session import BehaviorDataSession
+from allensdk.brain_observatory.behavior.behavior_ophys_session import BehaviorOphysSession
+import traceback
+import allensdk
+import datetime
+import sys
+import platform
+import pandas as pd
+import argparse
+import plotly.graph_objs as go
+import warnings
+from visual_behavior import database as db
+
+
+def get_cache():
+    MANIFEST_PATH = "//allen/programs/braintv/workgroups/nc-ophys/visual_behavior/2020_cache/production_cache/manifest.json"
+    cache = bpc.from_lims(manifest=MANIFEST_PATH)
+    return cache
+
+
+def is_ophys(behavior_session_id):
+    cache = get_cache()
+
+    behavior_session_table = cache.get_behavior_session_table()
+
+    return pd.notnull(behavior_session_table.loc[behavior_session_id]['ophys_session_id'])
+
+
+def bsid_to_oeid(behavior_session_id):
+    oeid = db.lims_query(
+        '''
+        select oe.id
+        from behavior_sessions
+        join ophys_experiments oe on oe.ophys_session_id = behavior_sessions.ophys_session_id
+        where behavior_sessions.id = {}
+        '''.format(behavior_session_id)
+    )
+    if isinstance(oeid, pd.DataFrame):
+        return oeid.iloc[0][0]
+    else:
+        return oeid
+
+
+def get_sdk_session(behavior_session_id, is_ophys):
```

Comment:
is there a reason not to have this in data_access and then just call it here?

Reply:
Moved to data_access.utilities